misc/determine-version.py

```diff
@@ -139,10 +140,10 @@ def version_cmp(a, b):
 # Find the full and abbreviated SHAs of the commit
 git = subprocess.Popen(["git", "rev-list", "-1", REV],
                        stdout=subprocess.PIPE)
-full_rev = git.stdout.readlines()[0].strip()
+full_rev = git.stdout.readlines()[0].decode().strip()
 git = subprocess.Popen(["git", "rev-list", "-1", "--abbrev-commit", REV],
                        stdout=subprocess.PIPE)
-abbrev_rev = git.stdout.readlines()[0].strip()
+abbrev_rev = git.stdout.readlines()[0].strip().decode()
```

Comment:
It seems more correct to decode and then strip here as well, and all places below. (?)

Reply:
OK, globally replaced.

Reply:
Blame it on that Trix guy.  He doesn't know any better.